src/screens/TopicPageScreen/index.js

```diff
@@ -164,10 +164,12 @@ const TopicPageScreen = (props) => {
       const resultTopicDetail = await getTopics(domain);
       if (resultTopicDetail.data) {
         const detail = resultTopicDetail.data.find((item) => item.name === domain);
-        setTopicDetail(detail);
-        handleFollowData(detail.is_followed_by);
-        setMemberCount(Number(detail?.followersCount));
-        setIsInitialLoading(false);
+        if (detail) {
+          setTopicDetail(detail);
+          handleFollowData(detail.is_followed_by);
+          setMemberCount(Number(detail?.followersCount));
+          setIsInitialLoading(false);
+        }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is also beneficial as it adds a check for `detail` before setting the state. This prevents potential issues if `detail` is null or undefined.
```diff
-        setTopicDetail(detail);
-        handleFollowData(detail.is_followed_by);
-        setMemberCount(Number(detail?.followersCount));
-        setIsInitialLoading(false);
+         if (detail) {
+           setTopicDetail(detail);
+           handleFollowData(detail.is_followed_by);
+           setMemberCount(Number(detail?.followersCount));
+           setIsInitialLoading(false);
+         }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->